halibot/halibot.py

```diff
@@ -28,6 +29,9 @@ def __init__(self, **kwargs):
 		self.log = logging.getLogger(self.__class__.__name__)
 
 		self.use_config = kwargs.get("use_config", True)
+		self.use_auth = kwargs.get("use_auth", True)
+
+		self.auth = HalAuth()
```

Comment:
Uhhhh probably. Will take a second look at. I added `auth.enabled` after `use_auth`, and forgot about the latter tbh :)